BuildResidentialHPXML/measure.rb

```diff
@@ -1013,6 +1013,12 @@ def arguments(model)
     arg.setDefaultValue(1)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_airflow_defect_ratio', false)
+    arg.setDisplayName('Heating System: Airflow Defect Ratio')
+    arg.setDescription("The airflow defect ratio, defined as (InstalledAirflow - DesignAirflow) / DesignAirflow, of the heating system. A value of zero means no airflow defect. Applies only to #{HPXML::HVACTypeFurnace}.")
```

Comment:
For all of these new arguments, let's add language "per ANSI/RESNET/ACCA Standard 310" to the description.